deploy/templates/20-no-show-proxy-dialog.pkla

```diff
@@ -0,0 +1,6 @@
+[No Show Proxy Dialog]
+Identity=unix-user:*
+Action=org.freedesktop.packagekit.system-network-proxy-configure
+ResultAny=no
+ResultInactive=no
+ResultActive=no
```

Comment:
@lel99999 All of these templates should be in the eod7 role, not at the top-level `templates` folder.

Reply:
Updated.